e2e-polybft/e2e/helpers_test.go

```diff
@@ -228,3 +228,21 @@ func isEventProcessed(t *testing.T, gatewayAddr types.Address,
 
 	return isProcessedAsNumber == 1
 }
+
+func isEventProcessedRollback(t *testing.T, gatewayAddr types.Address,
```

Comment:
This function is 90% the same as `isEventProcessed`. You can merge them in a single function that will receive an additional parameter called `processedEventFunction string`. Or if you want to keep it more simpler for usage: `isRollback bool`, and then in the function, based on that bool, determine the method name (`processedEventsRollback` or `processedEvents`).